common/api/core-common.api.md

```diff
@@ -10230,15 +10236,23 @@ export class ViewFlags {
 // @public
 export type ViewFlagsProperties = Mutable<NonFunctionPropertiesOf<ViewFlags>>;
 
+// @public
+export type ViewIdString = Id64String;
+
 // @public
 export interface ViewQueryParams extends EntityQueryParams {
     // (undocumented)
-    wantPrivate?: boolean;
+    readonly wantPrivate?: boolean;

```

Comment:
Motivation for breaking change?

Reply:
Good catch. I meant to take this out. At one point I changed this interface so you could use it to query ViewStores too. But I then decided that I’d make it so ViewStores are completely separate. This is a vestige of that that should be removed.

Reply:
fixed